### ISSUE TITLE
Main window and fonts suggestion 

### ISSUE THREAD

**Issue by mihha** [theterminalguy/floatcam#3]
Here comes my first suggestion

When I started your app for the first time, I was surprised that my firewall asked me to allow access to some external sites. When I checked the logs, I could see that this is because you are loading Google fonts from external sites. Because of GDPR or similar compliance initiatives, you should provide GDPR-compliant fonts. For example, Bunny Fonts are good drop in replacement fonts which are GDPR-compliant

https://fonts.bunny.net/about

Maybe it would be a good idea to either include those fonts into the package or use some other font. Your app won't be able to work correctly without internet connection and sometimes during the recording, that is exactly what is needed, in order to minimize annoyances 😀

Also, it would be great if you could provide a parameter to disable background animation in the main app window

To summarize:
1. remove the requirement to load google fonts and replace them with GDPR-compliant fonts 
2. disable or provide a way to disable background animation

Thank you in advance!

Regards,
Igor

**Comment by theterminalguy** [comment]
These are very useful feedback @mihha. 

I'll address them in the upcoming release and the main window will dock on the Windows 10 tray so it doesn't take the whole screen with the option to pop it back up if you want. 

Also, I'm considering adding a couple of features to floatcam. Can you [please help fill this out](https://forms.gle/J4BgqbDo1kYLxiPy7)? And share what features you'd like to see added

Thanks.

**Comment by mihha** [comment]
Done as requested

Thank you for useful app
